messages/src/maybe_known.rs

```diff
@@ -1,21 +1,22 @@
-use serde::{Deserialize, Serialize};
-
 /// Enum used to encapsulate `string-like` types which may
 /// have variants we haven't implement yet.
 ///
 /// Deserialization will be first attempted to the [`MaybeKnown::Known`] variant
 /// and then, if that fails, to the [`MaybeKnown::Unknown`] variant.
-#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
-#[serde(untagged)]
-pub enum MaybeKnown<T> {
-    Known(T),
-    Unknown(String),
-}
+// #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+// #[serde(untagged)]
+// pub enum MaybeKnown<T> {
+//     Known(T),
+//     Unknown(String),
+// }
+// implementation moved to shared_vcx::maybe_known::MaybeKnown
```

Comment:
Delete this file, please. I assume you wanted an easy way to revert if needed, but you'd have to modify the paths anyway, and that's probably the most cumbersome part 😅